### PR TITLE
Simplified TypeException error

### PR DESCRIPTION
**Pull request by arguiot**
# Fixes # 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:
- I improved readability and debugging by changing a TypeException error

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-php?pullRequest=738) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-php?pullRequest=738) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-php?pullRequest=738) it.</sub>

File: lib/mail/BatchId.php [hunk 1]
```diff
@@ -48,7 +48,7 @@ public function __construct($batch_id = null)
     public function setBatchId($batch_id)
     {
         if (!is_string($batch_id)) {
-            throw new TypeException('$batch_id must be of type string.');
+            throw new TypeException('$batch_id must be a string.');
```
Comment (peter279k) [inline lib/mail/BatchId.php]: I don't this exception messages are necessary to be changed.

If possible, can you fix all these related messages? I think some messages have the same problems, too :).

**Comment by arguiot** [comment]
@peter279k I changed a lot of TypeException errors, but I'm getting a build error that I'm not able to resolve, could you help me?

**Comment by peter279k** [comment]
Hi @arguiot, thank you for your reply. I've these same failure assertions before.

I think it's not the problems for your `TypeException` errors, it's about the internal Travis CI build issue.
Don't be worried about that :).

I also fork your latest batch `patch-1` and execute the Travis CI build in my account.
It works fine.

Please look at this [latest Travis CI build log](https://travis-ci.org/peter279k/sendgrid-php/builds/442308453).

**Comment by thinkingserious** [comment]
Hello @arguiot,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by arguiot** [comment]
@thinkingserious The link says that my swag order was proceeded. I guess that I’ve already filled the form. Is there any way I can see which address was recorded, just to be sure?

**Comment by childish-sambino** [comment]
Duplicate of https://github.com/sendgrid/sendgrid-php/pull/762

**Pull request closed without merging**